tutor/src/store/offering.ts

```diff
@@ -31,4 +31,8 @@ export const useAllOfferings = () => useSelector<OfferingSlice, Offering[]>(stat
     return selectors.selectAll(state)
 })
 
+export const useOfferingForId = (offeringId: ID) => useSelector<OfferingSlice, Offering | undefined>(state => {
+    return selectors.selectById(state, offeringId)
+})
+
```

Comment:
Used somewhere else?

Reply:
oh hah, I was going to use this but then saw the file was JS and using models so I just used that